dnd.sty

```diff
@@ -182,6 +182,7 @@
 
 % Load other modules of this package after all dependencies to avoid load order
 % conflicts (e.g., package options).
+\RequirePackage{lib/compat}       % compatibility definitions
```

Comment:
This looks good, but what about naming the file `dndobsolete.sty`? Then we can put all of our deprecated macros in here as well.

Reply:
We need to load this module before the other definitions.

A single module defining deprecated names is a great idea, because it will make maintenance easier. But we should load it after all the other modules so that it has access to the new definitions. We could use the after preamble/before document hooks to define the deprecated names, but may as well keep things simple by loading them at the end.